test/functional/lua/ui_spec.lua

```diff
@@ -134,7 +134,7 @@ describe('vim.ui', function()
 
   describe('open()', function()
     it('validation', function()
-      if not is_os('bsd') then
+      if is_ci('github') then
         matches('vim.ui.open: command failed %(%d%): { "[^"]+", "non%-existent%-file" }',
           exec_lua[[local _, err = vim.ui.open('non-existent-file') ; return err]])
```

Comment:
Was this broken somewhere (that lacks `xdg-open`) besides BSD?

Reply:
It's broken whenever there is a program that can open `non-existent-file`.